src/libstd/fs.rs

```diff
@@ -3369,6 +3393,7 @@ mod tests {
     }
 
     #[test]
+    #[cfg(not(target_os = "vxworks"))]
```

Comment:
FWIW it may just be best to not get the suite of tests on libstd working on vxworks. Maintaining these annotations is likely going to be burdensome and difficult to do, and there's not necessarily a lot to gain by gating on libstd tests since they're largely all OS-related anyway.